qemu/tests/netkvm_log_check.py

```diff
@@ -0,0 +1,73 @@
+import re
+from virttest import error_context
+from virttest import utils_net
+
+
+@error_context.context_aware
+def run(test, params, env):
+    """
+    Netkvm log checking using traceview:
+
+    1) Start the VM
+    2) Run `netkvmco.exe setparam 0 TXcapacity 64` in advance
+    3) Do the filters to grab the "Limit m_SGTableCapacity by 64 using
+       `TraceView.exe`"
+    4) Run `netkvmco.exe restart 0` or "NIC disabled-enabled"
+    5) Check the `TraceView.exe` monitor and get "Limit
+       m_SGTableCapacity by 64"
```

Comment:
Please check other test case how to write doc string, IMO, it would be more clear if we describe the test steps in natural language instead of list all command that the test cases execute.